conda/fetch.py

```diff
@@ -283,6 +283,17 @@ def fetch_pkg(info, dst_dir=None, session=None):
     path = join(dst_dir, fn)
 
     download(url, path, session=session, md5=info['md5'], urlstxt=True)
+    if info.get('sig'):
+        from conda.signature import verify
```

Comment:
Why is this import inside the conditional block, and not at the top of the file?